### PR TITLE
Added hiring and donation link to sponsorship page

### PR DESCRIPTION
**Pull request by egiurleo**
This PR adds links for sponsors to pay us for posting a job in our Slack or just making a corporate donation.

Screenshot 2024-06-21 at 10 39 25 AM

@luciagirasoles Please feel free to correct any of my React and styles here.

### CODE REVIEW

File: app/javascript/components/pages/SponsorUs.jsx [hunk 1]
```diff
@@ -11,12 +13,53 @@ const SponsorUs = () => (
...
+                        </p>
+                        <a
+                            href="https://buy.stripe.com/28o4gW9cZ7oKbwQ7sJ"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            <Button type="primary" className="max-w-[8rem] ">
+                                Post a job
+                            </Button>
+                        </a>
```
Comment (luciagirasoles) [inline app/javascript/components/pages/SponsorUs.jsx:41]: @egiurleo the job page link is sending to a stripe link

**Comment by luciagirasoles** [comment]
@egiurleo I added a background color the second section. Feel free to revert the changes if something is not good.

File: app/javascript/components/pages/SponsorUs.jsx (hunk 1 above)
Comment (luciagirasoles) [reply app/javascript/components/pages/SponsorUs.jsx:41]: maybe we can direct to an email while we work on the job page

Comment (egiurleo) [reply app/javascript/components/pages/SponsorUs.jsx:41]: We can talk about making a job page in the future though! I just want to do the lowest-lift version possible :)

Comment (luciagirasoles) [reply app/javascript/components/pages/SponsorUs.jsx:41]: I can understand. @egiurleo  I still need to confirm we are sending from that link to stripe too, is that correct?

Comment (egiurleo) [reply app/javascript/components/pages/SponsorUs.jsx:41]: Yup, that's exactly right!

**Comment by egiurleo** [comment]
@luciagirasoles The background color is beautiful! Thank you. Excellent addition.

**Comment by egiurleo** [comment]
If you're happy with the code, feel free to approve/merge whenever you're ready.

**Review by luciagirasoles — APPROVED** [review]

**Pull request merged**